core/src/main/java/com/ritense/valtimo/domain/user/UserSettings.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ *  Licensed under EUPL, Version 1.2 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.ritense.valtimo.domain.user;
+
+import org.hibernate.annotations.Type;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "user_settings")
+public class UserSettings {
+
+        @Id
+        @Column(name = "user_id", nullable = false)
+        private String userId;
+
+        @Type(type = "com.vladmihalcea.hibernate.type.json.JsonType")
+        @Column(name = "user_properties", columnDefinition = "JSON")
+        private String userProperties;
+
+        public UserSettings(){}
```

Comment:
I don't believe this empty constructor is needed anymore for JPA to work. If it does, please make it protected.